apis/projectcontour/v1alpha1/tlsciphers.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TLSCiphers holds a list of TLS ciphers
+type TLSCiphers []string
```

Comment:
I'm not sure about the maximum lengths for kubebuilder validations, but if we can fit the ciphersuites into kubebuilder tags, this whole file can just be the typedefs with kubebuilder tags instead.

Reply:
I think I got 'em in there, API server didn't care much. =)